pybv/io.py

```diff
@@ -64,6 +64,10 @@ def write_brainvision(data, sfreq, ch_names, fname_base, folder_out,
     fmt : str
         Binary format the data should be written as. Valid choices are
         'binary_float32' (default) and 'binary_int16'.
+    meas_date : str | None
+        The measurement date of the data specified as a string in the format:
+        "YYYYMMDDhhmmssuuuuuu". "u" stands for microseconds. If None, defaults
+        to '00000000000000000000'.
```

Comment:
why not allow for [ISO 8601 formatting](https://en.wikipedia.org/wiki/ISO_8601) which is a bit more human-readable?

Reply:
good point, I am now allowing `datetime.datetime` objects, I think this should cater to all needs :-)

See: https://github.com/bids-standard/pybv/pull/29/commits/d0467c2d087b77fc453de83094135c19902644ba